pipeline/transformer/transformer_sierra/src/test/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraMergeCandidatesTest.scala

```diff
@@ -271,15 +272,31 @@ class SierraMergeCandidatesTest
   }
 
   describe("Calm/Sierra harvest") {
-    it("Creates merge candidates from 035a from Calm/Sierra harvest") {
-      val ids = (1 to 5).map(_ => randomAlphanumeric(36).toString)
-      val calmMergeCandidates = ids map createCalmMergeCandidate
+    it("Assumes UUIDs in 035$a are from calm and attaches the merge candidate") {
+      def calmId = randomUUID.toString
 
+      val calmIds = (1 to 5).map(_ => calmId)
```

Comment:
Will this be an array of all the same IDs as `randomUUID` is only called once? Should it be
```scala
val calmIds = (1 to 5).map(_ => randomUUID.toString)
```

Reply:
As it's a `def` not a `val`, nope.

Reply:
ah, smart